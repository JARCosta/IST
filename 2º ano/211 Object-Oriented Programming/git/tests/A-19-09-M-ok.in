7
3
M1
2
GARRAFA
4
1
0
3
M1
2
GARRAFA
1
1
1
0
5
1
2
0
0
