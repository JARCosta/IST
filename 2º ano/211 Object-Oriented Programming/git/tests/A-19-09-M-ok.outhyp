Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Parceiros
7 - Gestão de Transacções
8 - Menu de Consultas
9 - Ver Saldo Global
0 - Sair
Escolha uma opção: Gestão de Transacções
1 - Visualizar
2 - Registar desagregação
3 - Registar venda
4 - Registar compra
5 - Receber pagamento de venda
0 - Sair
Escolha uma opção: Identificador do parceiro: Data limite de pagamento: Identificador do produto: Quantidade: Gestão de Transacções
1 - Visualizar
2 - Registar desagregação
3 - Registar venda
4 - Registar compra
5 - Receber pagamento de venda
0 - Sair
Escolha uma opção: Identificador da transacção: VENDA|0|M1|GARRAFA|4|50|50|2
Gestão de Transacções
1 - Visualizar
2 - Registar desagregação
3 - Registar venda
4 - Registar compra
5 - Receber pagamento de venda
0 - Sair
Escolha uma opção: Identificador do parceiro: Data limite de pagamento: Identificador do produto: Quantidade: Gestão de Transacções
1 - Visualizar
2 - Registar desagregação
3 - Registar venda
4 - Registar compra
5 - Receber pagamento de venda
0 - Sair
Escolha uma opção: Identificador da transacção: VENDA|1|M1|GARRAFA|1|15|15|2
Gestão de Transacções
1 - Visualizar
2 - Registar desagregação
3 - Registar venda
4 - Registar compra
5 - Receber pagamento de venda
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Parceiros
7 - Gestão de Transacções
8 - Menu de Consultas
9 - Ver Saldo Global
0 - Sair
Escolha uma opção: Gestão de Produtos
1 - Visualizar todos os produtos
2 - Visualizar todos os lotes
3 - Visualizar lotes fornecidos por parceiro
4 - Visualizar lotes de um dado produto
0 - Sair
Escolha uma opção: AA|2|43|B:1#C:1
B|1|1500
C|1|1500
CORTIÇA|1|1705
GARRAFA|15|0|ROLHA:11#VIDRO:5
PLASTICO|1|1041
ROLHA|2|2|AA:1#CORTIÇA:5#PLASTICO:1
SAL|100|500
VIDRO|1|185
Gestão de Produtos
1 - Visualizar todos os produtos
2 - Visualizar todos os lotes
3 - Visualizar lotes fornecidos por parceiro
4 - Visualizar lotes de um dado produto
0 - Sair
Escolha uma opção: AA|M1|2|43
B|M1|1|1500
C|M1|1|1500
CORTIÇA|M1|1|1705
PLASTICO|M1|1|1041
ROLHA|M1|2|2
SAL|S1|100|500
VIDRO|M1|1|185
Gestão de Produtos
1 - Visualizar todos os produtos
2 - Visualizar todos os lotes
3 - Visualizar lotes fornecidos por parceiro
4 - Visualizar lotes de um dado produto
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Parceiros
7 - Gestão de Transacções
8 - Menu de Consultas
9 - Ver Saldo Global
0 - Sair
Escolha uma opção: 