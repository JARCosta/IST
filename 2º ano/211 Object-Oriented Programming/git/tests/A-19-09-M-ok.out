Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Parceiros
7 - Gestão de Transacções
8 - Menu de Consultas
9 - Ver Saldo Global
0 - Sair
Escolha uma opção: Gestão de Transacções
1 - Visualizar
2 - Registar desagregação
3 - Registar venda
4 - Registar compra
5 - Receber pagamento de venda
0 - Sair
Escolha uma opção: Identificador do parceiro: Data limite de pagamento: Identificador do produto: Quantidade: Gestão de Transacções
1 - Visualizar
2 - Registar desagregação
3 - Registar venda
4 - Registar compra
5 - Receber pagamento de venda
0 - Sair
Escolha uma opção: Identificador da transacção: VENDA|0|M1|GARRAFA|4|233|233|2
Gestão de Transacções
1 - Visualizar
2 - Registar desagregação
3 - Registar venda
4 - Registar compra
5 - Receber pagamento de venda
0 - Sair
Escolha uma opção: Identificador do parceiro: Data limite de pagamento: Identificador do produto: Quantidade: Gestão de Transacções
1 - Visualizar
2 - Registar desagregação
3 - Registar venda
4 - Registar compra
5 - Receber pagamento de venda
0 - Sair
Escolha uma opção: Identificador da transacção: VENDA|1|M1|GARRAFA|1|115|115|2
Gestão de Transacções
1 - Visualizar
2 - Registar desagregação
3 - Registar venda
4 - Registar compra
5 - Receber pagamento de venda
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Parceiros
7 - Gestão de Transacções
8 - Menu de Consultas
9 - Ver Saldo Global
0 - Sair
Escolha uma opção: Gestão de Produtos
1 - Visualizar todos os produtos
2 - Visualizar todos os lotes
3 - Visualizar lotes fornecidos por parceiro
4 - Visualizar lotes de um dado produto
0 - Sair
Escolha uma opção: AA|2|0|B:1#C:1
B|1|1471
C|1|1471
CORTIÇA|1|1345
GARRAFA|115|0|ROLHA:11#VIDRO:5
PLASTICO|1|969
ROLHA|9|0|AA:1#CORTIÇA:5#PLASTICO:1
SAL|100|500
VIDRO|1|185
Gestão de Produtos
1 - Visualizar todos os produtos
2 - Visualizar todos os lotes
3 - Visualizar lotes fornecidos por parceiro
4 - Visualizar lotes de um dado produto
0 - Sair
Escolha uma opção: B|M1|1|1471
C|M1|1|1471
CORTIÇA|M1|1|1345
PLASTICO|M1|1|969
SAL|S1|100|500
VIDRO|M1|1|185
Gestão de Produtos
1 - Visualizar todos os produtos
2 - Visualizar todos os lotes
3 - Visualizar lotes fornecidos por parceiro
4 - Visualizar lotes de um dado produto
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Parceiros
7 - Gestão de Transacções
8 - Menu de Consultas
9 - Ver Saldo Global
0 - Sair
Escolha uma opção: 